_data/faq.yml

```diff
@@ -271,13 +271,13 @@
         en:
             title: Is it possible to edit an existing geocache log with c:geo?
             content: |
-                No, this is not possible and we do not have plans to implement this on short term.
-                Please use the geocaching website to edit your log after you posted it.
+                Yes, this is possible.
+                Tab on your log, a context menu will appear. Select "Edit log".
```

Comment:
English is not my native tongue, but shouldn't it read "tap on your log"? (with a "p" instead of a "b"?)

BTW: Thanks for helping us keeping our FAQ more up to date!

Reply:
English is also not my native to tongue, but I believe you are correct.